src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -4809,6 +4809,17 @@ in a slight, visible lag when scrolling complex pages.",
       "optional": true,
       "type": "boolean",
     },
+    Object {
+      "description": "A function from ISO8601 date string to return the desired time offset from UTC in minutes.
```

Comment:
"A function _that receives_ an ISO8601 date string _and returns_ the desired..."